cuda/common/include/pcl/cuda/cutil_inline_drvapi.h

```diff
@@ -86,7 +86,7 @@ inline int _ConvertSMVer2CoresDrvApi(int major, int minor)
 inline int cutilDrvGetMaxGflopsDeviceId()
 {
     CUdevice current_device = 0, max_perf_device = 0;
-    int device_count     = 0, sm_per_multiproc = 0;
+    int device_count     = 0, sm_per_multiproc;
```

Comment:
Variables should be never be uninitialized. In case the init value is not required it is usually a sign, that you can reduce the scope of this variable, as in this case (same for below).

Reply:
I'll keep that in mind.
I have also isolated declarations in this, as you suggested in another PR.